File: AdvancedDemo/PrimImageLoader/app/src/main/java/load/image/prim/com/primimageloader/MainActivity.java
package load.image.prim.com.primimageloader;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.LruCache;

import com.bumptech.glide.Glide;
import com.bumptech.glide.RequestManager;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        LruCache lruCache;
        RequestManager with = Glide.with(this);
    }
}

File: PrimFastCommon/app/src/main/java/com/prim/fast/MainActivity.kt
package com.prim.fast

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import cn.prim.http.lib_net.PrimHttp
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        sample_text.setOnClickListener {
//            var intent = Intent()
//            intent.setClass(this, TestHttpActivity::class.java)
//            startActivity(intent)
        }
    }
}
